src/routes/houses.routes.js

```diff
@@ -1,14 +1,23 @@
-import createNewHouseController from '../controllers/house.controllers';
+import { createNewHouseController, getHouseDetails } from '../controllers/house.controllers';
+
 const { StatusCodes } = require('http-status-codes');
 
-const createNewHouseRoutes = (router) => {
+const housesRoutes = (router) => {
   router.post('/create-new-house', async (req, res) => {
     const response = await createNewHouseController(req.body);
     if (!response || !response.status)
       return res.status(StatusCodes.INTERNAL_SERVER_ERROR).json({ status: 'server error' });
     if (response.status === 'invalid') return res.status(StatusCodes.BAD_REQUEST).json(response);
-    res.status(StatusCodes.CREATED).json(response);
+    return res.status(StatusCodes.CREATED).json(response);
+  });
+
+  router.get('/house/:id', async (req, res) => {
```

Comment:
The path could also be `house**s**/:id` to stay slightly more consistent with the names of other similar endpoints.